### DISCUSSION TITLE
When is layer freezing going to be enabled in yolov8?

### DISCUSSION THREAD

**Discussion by zz4fap** [ultralytics/ultralytics#2152] [Q&A]
Dear All,

When is layer freezing going to be enabled in yolov8?
We have tried the CLI "freeze" command, but it was to no avail.

Is there another way to freeze layers? 

Thanks and Kind Regards,

Felipe

**Comment by glenn-jocher** [comment]
@zz4fap hello Felipe,

Layer freezing should work with `--freeze` command.
